docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -314,9 +314,20 @@ scenario, and input draw.
   * - Person-time in each Alzheimer's cause state
     -
     -
-  * - Test counts (CSF, PET, BBBM)
-    - By diagnosis provided (for BBBM tests)
+  * - Existing (CSF/PET) test counts
     -
+    -
+  * - BBBM test counts
+    - * Diagnosis provided
+    -
+  * - Existing (CSF/PET) test-eligibile simulants
+    - 
+    - Number of simulants who are eligible for existing (CSF/PET) testing, based on the :ref:`PET/CSF eligibility requirements <petcsf_requirements>` (list in step 1).
+      Will be used to check that eligible simulants * test rate = test count, for each location-specific test rate.
+  * - BBBM test-eligibile simulants
+    - 
+    - Number of simulants who are eligible for BBBM testing, based on the :ref:`BBBM eligibility requirements <bbbm_requirements>` (list in step 1).
```

Comment:
Does it matter that we'll be recording this number twice per year and adding the results? E.g., will we have double-counting issues or something (which I think would get worse if we had a shorter step size)? We could imagine recording "eligible person-time" instead, where the count would get multiplied by the step size, but I'm not sure if that's what we want either.